openedx/core/djangoapps/course_date_signals/utils.py

```diff
@@ -21,11 +20,6 @@ def get_expected_duration(course):
 
     access_duration = MIN_DURATION
 
-    verified_mode = CourseMode.verified_mode_for_course(course=course, include_expired=True)
-
-    if not verified_mode:
-        return None
```

Comment:
This felt like an odd check in the middle of a utility method to get course duration. I moved it up a level to get_user_course_duration, which has a similar check on the user enrollment.